### PR TITLE
Feature/eslint/qgrid ngx examples

### PR DESCRIPTION
**Pull request by Dalantren**

### CODE REVIEW

File: packages/qgrid-ngx-examples/src/examples/action-bar-template/example-action-bar-template.component.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<q-grid-actions>
 		<q-grid-action>
 			<ng-template for="trigger"
-						 let-$action>
+						let-$action>
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/action-bar-template/example-action-bar-template.component.html:9]: Shouldn't tab width be 2? This here is way beyond 2.

File: packages/qgrid-ngx-examples/src/examples/action-bar-template/example-action-bar-template.component.html [hunk 2]
```diff
@@ -15,12 +15,12 @@
 		</q-grid-action>
 		<q-grid-action>
 			<ng-template for="trigger"
-						 let-$action>
+						let-$action>
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/action-bar-template/example-action-bar-template.component.html:18]: Tab width.

File: packages/qgrid-ngx-examples/src/examples/column-array-basic/example-column-array-basic.component.html [hunk 3]
```diff
@@ -4,16 +4,16 @@
 	<q-grid-columns generation="deep">
 		<q-grid-column [width]="170"></q-grid-column>
 		<q-grid-column key="customTemplate"
-					   [viewWidth]="200"
-					   title="Custom Template"
-					   type="array">
+						[viewWidth]="200"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/column-array-basic/example-column-array-basic.component.html:7]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/column-autocomplete-basic/example-column-autocomplete-basic.component.html [hunk 4]
```diff
@@ -3,58 +3,58 @@
 		editMode="cell">
 	<q-grid-columns>
 		<q-grid-column key="number"
-					   title="Number"
-					   type="number"
-					   editor="autocomplete"
-					   [editorOptions]="numberObservableFetchOptions">
+						title="Number"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/column-autocomplete-basic/example-column-autocomplete-basic.component.html:6]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/column-id-basic/example-column-id-basic.component.html [hunk 5]
```diff
@@ -1,53 +1,53 @@
...
-					   editor="date"
-		               type="id">
+							title="Date"
+							editor="date"
+							type="id">
 		</q-grid-column>
 		<q-grid-column key="text"
-		               title="Text"
-		               type="id">
+									title="Text"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/column-id-basic/example-column-id-basic.component.html:21]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/column-image-basic/example-column-image-basic.component.html [hunk 6]
```diff
@@ -1,41 +1,41 @@
...
-		               title="Invalid"
-		               type="image">
+									title="Invalid"
+									type="image">
 		</q-grid-column>
 		<q-grid-column key="valid"
-		               title="Valid"
-		               type="image">
+									title="Valid"
+									type="image">
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/column-image-basic/example-column-image-basic.component.html:11]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/column-list-loop/example-column-list-loop.component.html [hunk 7]
```diff
@@ -1,11 +1,11 @@
 <q-grid [caption]="title"
-        [rows]="rows | async">
+				[rows]="rows | async">
 	<q-grid-columns>
 		<q-grid-column *ngFor="let column of columns"
-		               [key]="column.key"
-		               [title]="column.title">
+									[key]="column.key"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/column-list-loop/example-column-list-loop.component.html:5]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/column-password-basic/example-column-password-basic.component.html [hunk 8]
```diff
@@ -1,52 +1,52 @@
...
 		<q-grid-column key="date"
-		               title="Date"
-		               type="password">
+									title="Date"
+									type="password">
 		</q-grid-column>
 		<q-grid-column key="null"
-		               title="Null"
-		               type="password">
+									title="Null"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/column-password-basic/example-column-password-basic.component.html:18]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/column-url-basic/example-column-url-basic.component.html [hunk 9]
```diff
@@ -1,37 +1,37 @@
...
 		<q-grid-column key="invalid"
-		               title="Invalid"
-		               type="url">
+									title="Invalid"
+									type="url">
 		</q-grid-column>
 		<q-grid-column key="valid"
-		               title="Valid"
-		               type="url">
+									title="Valid"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/column-url-basic/example-column-url-basic.component.html:10]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/copy-value-from-cell/example-copy-value-from-cell.component.html [hunk 10]
```diff
@@ -7,14 +7,14 @@
 		[rows]="rows">
 	<q-grid-columns>
 		<q-grid-column key="dollar"
-		               title="Dollar"
-		               type="currency">
+									title="Dollar"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/copy-value-from-cell/example-copy-value-from-cell.component.html:10]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/data-row-delete-selection/example-data-row-delete-selection.component.html [hunk 11]
```diff
@@ -1,19 +1,19 @@
...
-  </q-grid-columns>
-</q-grid>
\ No newline at end of file
+				[rows]="rows | async"
+				selectionMode="multiple"
+				selectionUnit="row">
+	<q-grid-columns generation="deep">
+		<q-grid-column type="row-options"
+									key="rowOptions"
+									[width]="150"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/data-row-delete-selection/example-data-row-delete-selection.component.html:8]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/define-column-async/example-define-column-async.component.html [hunk 12]
```diff
@@ -1,18 +1,18 @@
 <q-grid [caption]="title"
-        [model]="gridModel">
+				[model]="gridModel">
 	<q-grid-columns>
 		<q-grid-column key="number"
-		               title="#"
-		               [width]="60"
-		               type="id">
+									title="#"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/define-column-async/example-define-column-async.component.html:5]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/define-column-basic/example-define-column-basic.component.html [hunk 13]
```diff
@@ -3,22 +3,22 @@
...
+									title="#"
+									[width]="60"
+									type="id">
 		</q-grid-column>
 		<q-grid-column key="symbol"
-		               title="Symbol"
-		               [width]="80"
-		               type="text">
+									title="Symbol"
+									[width]="80"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/define-column-basic/example-define-column-basic.component.html:12]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/define-column-hybrid/example-define-column-hybrid.component.html [hunk 14]
```diff
@@ -3,22 +3,22 @@
 		[columns]="columns">
 	<q-grid-columns>
 		<q-grid-column key="number"
-		               title="should be html №"
-					   [width]="90"
-		               type="id">
+									title="should be html №"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/define-column-hybrid/example-define-column-hybrid.component.html:6]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/edit-row-custom/example-edit-row-custom.component.html [hunk 15]
```diff
@@ -1,32 +1,32 @@
...
 
-    <q-grid-actions>
-        <q-grid-action icon="edit"
-                       title="Open Pane"
-                       [command]="openPane">
-        </q-grid-action>
-    </q-grid-actions>
+		<q-grid-actions>
+				<q-grid-action icon="edit"
+											title="Open Pane"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/edit-row-custom/example-edit-row-custom.component.html:10]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/group-column-basic/example-group-column-basic.component.html [hunk 16]
```diff
@@ -1,59 +1,59 @@
...
 		<q-grid-column title="Birthday"
-		               key="birthday">
+									key="birthday">
 		</q-grid-column>
 		<q-grid-column title="Contact Info">
 			<q-grid-column title="Email"
-			               key="email"
-			               path="contact.email">
+										key="email"
+										path="contact.email">
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/group-column-basic/example-group-column-basic.component.html:23]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/interaction-mode-detached/example-interaction-mode-detached.component.html [hunk 17]
```diff
@@ -1,18 +1,18 @@
...
-</q-grid>
\ No newline at end of file
+				interactionMode="detached"
+				[rows]="rows | async">
+				<q-grid-columns generation="deep">
+								<q-grid-column key="test"
+															title="Test Detached"
+															[index]="0"
+															[canFilter]="false"
+															[canSort]="false">
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/interaction-mode-detached/example-interaction-mode-detached.component.html:9]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/interaction-mode-readonly/example-interaction-mode-readonly.component.html [hunk 18]
```diff
@@ -1,16 +1,16 @@
...
-        </q-grid-columns>
-</q-grid>
\ No newline at end of file
+				interactionMode="readonly"
+				[rows]="rows | async">
+				<q-grid-columns generation="deep">
+								<q-grid-column key="test"
+															title="Test Detached"
+															[index]="0">
+												<ng-template for="body">
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/interaction-mode-readonly/example-interaction-mode-readonly.component.html:8]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/look-atoms-customized/example-look-atoms-customized.component.html [hunk 19]
```diff
@@ -1,38 +1,38 @@
...
 				</div>
 			</ng-template>
 		</q-grid-column>
 		<q-grid-column key="number"
-		               type="id"
-		               title="#"
-		               [width]="60"
-		               [canFilter]="false">
+									type="id"
+									title="#"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/look-atoms-customized/example-look-atoms-customized.component.html:17]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/persistence-server/example-persistence-server.component.html [hunk 20]
```diff
@@ -1,15 +1,15 @@
...
+				[rows]="rows | async"
+				[model]="gridModel"
+				id="persistence-server">
 	<q-grid-toolbar>
 		<ng-template for="top">
 			<div class="persistence-server-toolbar">
 				<mat-form-field>
 					<mat-select [(value)]="currentUser"
-					            placeholder="Select user">
+											placeholder="Select user">
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/persistence-server/example-persistence-server.component.html:10]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/pivot-column-basic/example-pivot-column-basic.component.html [hunk 21]
```diff
@@ -1,14 +1,14 @@
 <q-grid [caption]="title"
-        [rows]="rows | async"
-        [pivotBy]="['bondingType', 'groupBlock']">
+				[rows]="rows | async"
+				[pivotBy]="['bondingType', 'groupBlock']">
 	<q-grid-columns>
 		<q-grid-column key="name"
-		               [width]="90"
-		               title="Name">
+									[width]="90"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/pivot-column-basic/example-pivot-column-basic.component.html:6]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/pivot-column-cohort/example-pivot-column-cohort.component.html [hunk 22]
```diff
@@ -1,18 +1,18 @@
...
+				[pivotBy]="['bondingType', 'groupBlock']">
 	<q-grid-columns>
 		<q-grid-column title="Name">
 			<q-grid-column key="number"
-			               title="Number">
+										title="Number">
 			</q-grid-column>
 			<q-grid-column key="symbol"
-			               title="Symbol">
+										title="Symbol">
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/pivot-column-cohort/example-pivot-column-cohort.component.html:10]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/pivot-column-template/example-pivot-column-template.component.html [hunk 23]
```diff
@@ -1,25 +1,25 @@
 <q-grid [caption]="title"
-        [rows]="rows | async"
-        [pivotBy]="['bondingType', 'groupBlock']">
+				[rows]="rows | async"
+				[pivotBy]="['bondingType', 'groupBlock']">
 	<q-grid-columns>
 		<q-grid-column key="name"
-		               [width]="90"
-		               title="Name">
+									[width]="90"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/pivot-column-template/example-pivot-column-template.component.html:6]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/scroll-virtual-details/example-scroll-virtual-details.component.html [hunk 24]
```diff
@@ -1,42 +1,43 @@
...
-		               [width]="70">
+									type="id"
+									title="#"
+									[width]="70">
 		</q-grid-column>
 		<q-grid-column key="symbol"
-		               title="Symbol"
-		               [width]="90">
+									title="Symbol"
+									[width]="90">
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/scroll-virtual-details/example-scroll-virtual-details.component.html:12]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/scroll-virtual-edit/example-scroll-virtual-edit.component.html [hunk 25]
```diff
@@ -1,19 +1,19 @@
...
-    </q-grid-columns>
+				scrollMode="virtual"
+				[rows]="rows | async"
+				editMethod="batch">
+		<q-grid-columns generation="deep">
+				<q-grid-column key="rowIndicator"
+											type="row-indicator">
+				</q-grid-column>
+				<q-grid-column key="rowOptions"
+											type="row-options">
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/scroll-virtual-edit/example-scroll-virtual-edit.component.html:10]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/scroll-virtual-height/example-scroll-virtual-height.component.html [hunk 26]
```diff
@@ -1,14 +1,14 @@
...
-        <q-grid-columns generation="deep">
-                <q-grid-column type="row-number"
-                               key="rowNumber">
-                </q-grid-column>
-        </q-grid-columns>
+				scrollMode="virtual"
+				[rows]="rows | async">
+				<q-grid-columns generation="deep">
+								<q-grid-column type="row-number"
+															key="rowNumber">
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/scroll-virtual-height/example-scroll-virtual-height.component.html:6]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/scroll-virtual-selection/example-scroll-virtual-selection.component.html [hunk 27]
```diff
@@ -1,12 +1,12 @@
...
-        selectionMode="range">
-        <q-grid-columns generation="deep">
-                <q-grid-column type="select" key="select"></q-grid-column>
-        </q-grid-columns>
+				scrollMode="virtual"
+				[rows]="rows | async"
+				selectionUnit="mix"
+				selectionMode="range">
+				<q-grid-columns generation="deep">
+								<q-grid-column type="select" key="select"></q-grid-column>
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/scroll-virtual-selection/example-scroll-virtual-selection.component.html:7]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/scroll-virtual-style/example-scroll-virtual-style.component.html [hunk 28]
```diff
@@ -1,16 +1,16 @@
...
-                               [width]="100"
-                               key="number">
-                </q-grid-column>
-        </q-grid-columns>
+				scrollMode="virtual"
+				[styleCell]="styleCell"
+				[rows]="rows | async">
+				<q-grid-columns generation="deep">
+								<q-grid-column type="row-number"
+															[width]="100"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/scroll-virtual-style/example-scroll-virtual-style.component.html:7]: Check indentation here.

File: packages/qgrid-ngx-examples/src/examples/size-column-basic/example-size-column-basic.component.html [hunk 29]
```diff
@@ -1,21 +1,21 @@
...
+								<q-grid-column title="Width 120px"
+															key="number"
+															[width]="120">
+								</q-grid-column>
+								<q-grid-column title="Width 15%"
+															key="symbol"
+															width="15%">
+								</q-grid-column>
+								<q-grid-column title="Width 35%"
+															key="name"
```
Comment (FinalButterfly) [inline packages/qgrid-ngx-examples/src/examples/size-column-basic/example-size-column-basic.component.html:13]: Check indentation here.

**Review by FinalButterfly — APPROVED** [review]

**Review by klumba12 — APPROVED** [review]

**Comment by Dalantren** [comment]
@FinalButterfly I checked your comments, they all is fine now.

**Pull request merged**